nomenclature/validation_schemas/region_mapping_schema.yaml

```diff
@@ -0,0 +1,52 @@
+$schema: 'https://json-schema.org/draft/2020-12/schema'
+title: RegionAggregationMapping
+description: >
+  This schema is used to do the first stage of validation for yaml region
+  mapping files.
+type: object
+properties:
+  model:
+    title: Model
+    type: string
+  native_regions:
+    title: Native Regions
+    type: array
+    items:
+      $ref: '#/definitions/NativeRegion'
+    minItems: 1
+  common_regions:
+    title: Common Regions
+    type: array
+    items:
+      $ref: '#/definitions/CommonRegion'
+    minItems: 1
+required:
+  - model
+additionalProperties: false
+definitions:
+  NativeRegion:
+    title: NativeRegion
+    oneOf:
+      - type: object
+        patternProperties:
+          ^.+$:
```

Comment:
Is it possible to add a comment here? Maybe "the key of the dictionary is the native-region name"...